ledger-core/application/testwalletapi/statemachine/statemachine.go

```diff
@@ -63,6 +63,7 @@ func (s *SMTestAPICall) GetStateMachineDeclaration() smachine.StateMachineDeclar
 }
 
 func (s *SMTestAPICall) migrationDefault(ctx smachine.MigrationContext) smachine.StateUpdate {
+	ctx.Log().Trace("Resend message")
```

Comment:
please delete this trace, convlogtxt logs every step of SM. I don't understand why you need this?

Reply:
It adds more clarity when you need to troubleshoot

Reply:
I disagree, every step and every jump is logged in a defined format. I also add some troubleshooting points with spacers ("\n\n"), but don't leave them in commits.